integration-tests/cypress/workbench/GanttTestHelper.ts

```diff
@@ -0,0 +1,131 @@
+/*******************************************************************************
+ * Copyright (c) 2024 Obeo.
+ * This program and the accompanying materials
+ * are made available under the terms of the Eclipse Public License v2.0
+ * which accompanies this distribution, and is available at
+ * https://www.eclipse.org/legal/epl-2.0/
+ *
+ * SPDX-License-Identifier: EPL-2.0
+ *
+ * Contributors:
+ *     Obeo - initial API and implementation
+ *******************************************************************************/
+
+import { Project } from '../pages/Project';
+import { Projects } from '../pages/Projects';
+import { isCreateProjectFromTemplateSuccessPayload } from '../support/server/createProjectFromTemplateCommand';
+import { Explorer } from './Explorer';
+
+export class GanttTestHelper {
```

Comment:
To be constitent with the other class, please name it `Gantt`

Reply:
I known, I know but I dislike those simple names
Changed